securedrop/requirements/update_python_dependencies

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Usage: ./update-python-dependencies
```

Comment:
@redshiftzero I might have a better workflow than the one described in that discussion. We should talk IRL about it.